test/01_typedef.jl

```diff
@@ -1,7 +1,8 @@
 module TestTypeDef
 using Base.Test
 using CategoricalArrays
-using CategoricalArrays: DefaultRefType, level,  reftype, leveltype, catvalue, iscatvalue
+using CategoricalArrays: DefaultRefType, level, reftype, leveltype, catvalue, iscatvalue
+using CategoricalArrays: CategoricalPool
```

Comment:
Put this on the previous line.

Reply:
fixed, although it produces overlong line

Reply:
Yeah, but who said we are not allowed to wrap lines? ;-)

Reply:
👍